### PR TITLE
Merge develop 8.2 17 12 19

### PR DESCRIPTION
**Pull request by montegoulding**
Resolution gist https://gist.github.com/a53f8a7a7d83c11dc89da14741250e27

### CODE REVIEW

File: engine/rsrc/emscripten-startup-template.livecodescript [hunk 1]
```diff
@@ -21,7 +21,7 @@ constant kExtensionListFile = "/boot/extensions/extensions.txt"
 on startup
    local tError, tFolder, tFile, tByteString, tByte
    local tStackName, tStackData, tStackHashString
-   local tModuleList, tModuleName
+   local tModuleList, tModuleName, tAuxiliaryList
    try
```
Comment (livecodepanos) [inline engine/rsrc/emscripten-startup-template.livecodescript:25]: This local var (`tAuxiliaryList`) is not used at all

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 3eed3bb

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 3eed3bb69f28100df15ba2cd8b58f0e430b02b53

**Comment by livecode-vulcan** [comment]
😎 test success 3eed3bb69f28100df15ba2cd8b58f0e430b02b53

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1098)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1114)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1088)
* try-community-universal-ios-iphoneos11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.2/builds/41)
* try-community-universal-ios-iphonesimulator11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.2/builds/41)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1093)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1115)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1098)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1114)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1097)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1107)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1130)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1093)

**Pull request merged**